src/test/docs-test.ts

```diff
@@ -73,4 +73,49 @@ describe('docs test getSwaggerFiles', () => {
 
     assert.deepStrictEqual(res.stable, ['src/test/readmes/Microsoft.Kusto/stable/2022-07-07/kusto.json'])
   })
+
+  it.skip('Test getSwaggerFiles for api management.', () => {
```

Comment:
why is this skipped?

Reply:
That's for local test. When we want to test it with some local spec repo.